waiter/integration/waiter/basic_test.clj

```diff
@@ -380,7 +380,7 @@
   (testing-using-waiter-url
     (run-backend-proto-service-test waiter-url "h2c" 1 "http" "HTTP/2.0")))
 
-(deftest ^:parallel ^:integration-fast test-h2-backend-proto-service
+(deftest ^:parallel ^:integration-fast ^:explicit test-h2-backend-proto-service
```

Comment:
Did we create an issue for this with a trace to the logs?

Reply:
Yep, sent you the link

Reply:
Appears to be kubernetes issue, let's discuss this tomorrow.

Reply:
Sounds good, do you want me to un-mark `:explicit`?

Reply:
Yes, but want to figure out what's the best way to tackle this k8s test flakes due to out of resource issues.